packages/core/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -121,11 +121,13 @@ class AddressDetails extends React.Component {
                                                         ? localize('First line of address*')
                                                         : localize('First line of address')
                                                 }
+                                                maxLength={255}
                                                 placeholder={localize('First line of address')}
                                             />
                                             <InputField
                                                 name='address_line_2'
                                                 label={localize('Second line of address')}
+                                                maxLength={255}
```

Comment:
why allow 255 if the max is 70?

Reply:
I think we should let the `validation` handle the `min` and `max` values instead of setting it on the field prop.

Because if we set `maxLength` on the input field, there's no point of having the `length` validation for max since we're preventing the user character input `onKeyPress` after the `70th` character anyways.

For example, if the user copies an address that is over 70 characters into the field, the characters will get truncated instead of keeping the strings and displaying the length validation message.

Reply:
sure, then we should remember to update the rest of the app to match this then :D